### DISCUSSION TITLE
browser automated tests

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1306] [Q&A] [answered]
Trying to setup automated browser point-and-click testing.

Running the cypress example for the `Wizard` app from the docs, but cypress is giving an error

`cy.locate is not a function`

also tried `location()` and `find()`.

cypress version is 9.5.2

**Comment by mturoci** [comment, marked as answer]
Have a look at [Wave GH test dir](https://github.com/h2oai/wave/tree/master/test) - this is what our docs use as a reference (although not saying it explicitly, need to fix it). In that dir, there is a [commands.js](https://github.com/h2oai/wave/blob/master/test/cypress/support/commands.js) which registers the custom `locate` function that is failing for you. You have to register it within your cypress installation folder as well in order to use it.
